src/BlazorUI/Bit.BlazorUI/Components/Inputs/TextField/BitTextField.scss

```
@@ -7,10 +7,21 @@
     font-weight: 400;
     position: relative;
     box-sizing: border-box;
-    font-size: spacing(1.75);
     font-family: $tg-font-family;
+    font-size: var(--bit-tfl-font-size);
+    // The medium size is the default one, so its values sit on the root instead of only on .bit-tfl-md.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)